### PR TITLE
israel

### PR DESCRIPTION
**Pull request by adeoyebrass**
help

### CODE REVIEW

**Comment by dro-1** [comment]
Delete the script file in the root directory

File: scripts/israel.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var text="hello world,Am ";
+var fullname="Israel ";
+var word="from ";
+var company="HNGi7 ";
+var id="HNG-04195 ";
+var text1="using ";
+var language="javascript language ";
+var stage="for stage 2 task email is,";
+var email="adeoyefemi3@gmail.com";
+console.log(text+fullname+word+company+id+text1+language+stage+email);
```
Comment (alexismajya) [inline scripts/israel.js:10]: wrong output. 
1 text = it should be " Hello World, this is"
2. no spaces!!
3. company = should be "HNGi7 ID"
4 .email comes before text1 and add "and" after id

Comment (alexismajya) [reply scripts/israel.js:10]: language should be "javascript" not javascript language

**Review by alexismajya — CHANGES_REQUESTED** [review]
wrong output

**Comment by simply-alliv** [comment]
@dro-1 Sorry for barging in but the script itself should also update be updated because it will fail the test.

@adeoyebrass See how the [goal/output](https://github.com/hng-teamfury/team-fury-1/blob/dev/README.md#the-goal) of the script should look like and make sure that yours follows suit.

File: scripts/israel.js (hunk 1 above)
Comment (alexismajya) [reply scripts/israel.js:10]: stage= "for stage 2 task"

**Comment by Rubylena** [comment]
am closing this, when your ready

**Comment by adeoyebrass** [comment]
> Delete the script file in the root directory

how please

**Comment by adeoyebrass** [comment]
> am closing this, when your ready

am ready pls

**Pull request closed without merging**